src/app/games/epaminondas/tests/EpaminondasAttackHeuristic.spec.ts

```diff
@@ -53,44 +47,46 @@ describe('AttackEpaminondasHeuristic', () => {
             [O, O, O, O, O, O, O, O, O, O, O, _, _, O],
             [O, O, O, O, O, O, O, O, O, O, O, _, O, _],
         ];
-        const lesserState: EpaminondasState = new EpaminondasState(lesserBoard, 1);
+        const weakerState: EpaminondasState = new EpaminondasState(weakerBoard, 1);
         HeuristicUtils.expectSecondStateToBeBetterThanFirstFor(heuristic,
-                                                               lesserState, MGPOptional.empty(),
-                                                               greaterState, MGPOptional.empty(),
+                                                               weakerState, MGPOptional.empty(),
+                                                               strongerState, MGPOptional.empty(),
                                                                Player.ONE);
```

Comment:
p-e ré-expliquer ce test du point de vue de ZERO, plutôt que dire "should prefer to advance, so when the opponent does it's a bad board!" :D ?

Reply:
oui c'est mieux !